onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1120,6 +1120,7 @@ TEST(InferenceSessionTests, TestTruncatedSequence) {
   ASSERT_TRUE(model_file_stream.good());
   ModelProto model_proto;
   Model::Load(model_file_stream, &model_proto);
+  model_file_stream.close();
```

Comment:
You may also try this code:
```cpp
  ONNX_NAMESPACE::ModelProto proto;
  int test_data_pb_fd;
  common::Status st = Env::Default().FileOpenRd(model_file_stream, test_data_pb_fd);
   ASSERT_TRUE(st.IsOK());
  google::protobuf::io::FileInputStream f(test_data_pb_fd);
  f.SetCloseOnDelete(true);
  ASSERT_TRUE(proto.ParseFromZeroCopyStream(&f));
```
Reading from FD is faster than reading from istream.  Anyway, this is testing code, performance is not a matter.